modules/setup-openshift-oidc-provider.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-using-short-lived-tokens.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="setup-openshift-oidc-provider_{context}"]
+= Setup OpenShfit OIDC provider
```

Comment:
Gerund

Not sure how to word this exactly, so not sure if the following example is correct: 

```suggestion
= Setting up an OIDC provider for {ocp}
```